dev/build_google_component_image.sh

```diff
@@ -173,14 +173,16 @@ function create_component_prototype_disk() {
       --command="sudo DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade && sudo apt-get autoremove -y"
   fi
 
-  echo "`date`: Deleting '$BUILD_INSTANCE' but keeping disk"
+  echo "`date`: Setting auto-delete behavior '$BUILD_INSTANCE' disk to false"
   gcloud compute instances set-disk-auto-delete $BUILD_INSTANCE \
     --project $BUILD_PROJECT \
     --account $ACCOUNT \
     --zone $ZONE \
     --no-auto-delete \
     --disk $BUILD_INSTANCE
 
+  sleep 20
```

Comment:
If the script made it this far, then the long parts (boot the VMs & successfully run the install scripts) must have passed. 

There is already a lot of stateful behavior with global flags for both the target VM & the `trap` statements in that script making it hard to reason about, adding another, especially one that implemented incorrectly will leave artifacts around causing future builds to fail, feels like more trouble than it is worht.